src/nbl/video/COpenGLRenderpassIndependentPipeline.h

```diff
@@ -36,7 +36,8 @@ class COpenGLRenderpassIndependentPipeline final : public IGPURenderpassIndepend
             _vertexInputParams, _blendParams, _primAsmParams, _rasterParams
             ),
             IOpenGLPipeline(_ctxCount, _ctxID, _GLnames, _binaries),
-            m_stagePresenceMask(0u)
+            m_stagePresenceMask(0u),
+            m_baseInstanceUniformIDs(core::make_refctd_dynamic_array<decltype(m_baseInstanceUniformIDs)>(_ctxCount))
```

Comment:
this hurts... can you not slap on an extra 4 bytes to the uniform variables state? and use it out of there?